src/Wallabag/CoreBundle/Controller/EntryController.php

```diff
@@ -434,4 +436,32 @@ private function checkIfEntryAlreadyExists($entry)
     {
         return $this->get('wallabag_core.entry_repository')->findByUrlAndUserId($entry->getUrl(), $this->getUser()->getId());
     }
+
+    /**
+     * Share entry content.
+     *
+     * @param Entry $entry
+     *
+     * @Route("/share/{uuid}", requirements={"uuid" = ".+"}, name="share")
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function shareEntryAction(Entry $entry)
+    {
+        return $this->render(
+            '@WallabagCore/themes/share.html.twig',
+            array('entry' => $entry)
+        );
+    }
```

Comment:
Since we are sending a real public response, we might want to use a Response with some cache header, etc ...